packages/cxl-ui/src/components/cxl-marketing-nav.js

```diff
@@ -273,7 +280,27 @@ export class CXLMarketingNavElement extends LitElement {
         );
 
         // Populate.
-        contextMenu.items = this._createContextMenuItems(menuItem.children);
+        const contextMenuItems = this._createContextMenuItems(menuItem.children);
+        contextMenu.addEventListener('opened-changed', () => {
+          contextMenu.items = contextMenuItems;
+          const listBox = document.querySelector('vaadin-context-menu-list-box');
+          setTimeout(() => {
+            const { width } = getComputedStyle(listBox);
+            if (menuItem.description) {
+              const el = document.createElement('vaadin-context-menu-item');
+              el.style.minWidth = '256px';
```

Comment:
For text stuff, it usually makes sense to use `ch` units.

But I'm not sure min-width is a good idea wrt to x-device compatibility. Let's deal with an iteration when real content setup demands it at one point.

Reply:
I agree with you on using `ch` units. The code needs to be refactored to account for a switch in device orientation. I can work on that.